poly-synth/src/poly-synth.js

```diff
@@ -72,13 +90,101 @@ class PolySynth {
       filterDecay: 1,
       filterSustain: this.minSustain,
       filterRelease: this.minRelease,
-      filterDetuneAmount: 1
+      filterDetuneAmount: 1,
     };
 
+    // Each voice is connected to |this.voiceOutput_|.
+    this.voiceOutput_ = new GainNode(this.context_);
+
+    // The output of |this.voiceOutput_| is processed by three effects
+    // bitcrusher, feedback delay, and a convolver.
+    let feedbackInputNode = new GainNode(this.context_);
+    let feedbackOutputNode = this.createFeedbackGraph_(feedbackInputNode);
+
+    let bitcrusherInputNode = new GainNode(this.context_);
+    let bitcrusherOutputNode = this.createBitcrusherGraph_(bitcrusherInputNode);
+
+    let convolverInputNode = new GainNode(this.context_);
+    let convolverOutputNode = this.createConvolverGraph_(convolverInputNode);
+
     // The client is responsible for connecting |this.output| to a destination.
     this.output = new GainNode(this.context_);
+
+    // The feedback node occurs prior to the other effects so that the output
+    // of the feedback can be modulated by these components.
+    this.voiceOutput_.connect(feedbackInputNode);
+    feedbackOutputNode.connect(bitcrusherInputNode);
+    bitcrusherOutputNode.connect(convolverInputNode);
+    convolverOutputNode.connect(this.output);
+  }
+
+  createFeedbackGraph_(inputNode) {
+    // The sum of the gain of the wet and dry signal is always 1, and the output
+    // always routes through both nodes.
+    this.feedbackWetSignal_ =
+        new GainNode(this.context_, {gain: this.feedbackWetness});
+    this.feedbackDrySignal_ =
```

Comment:
I find these names confusing when thinking about the graph.  Maybe feedbackWetSignalGain and feedbackDrySignalGain?  That tells me these are the gains for the different signals, not the signals themselves.Maybe.